hack/vendor.sh

```diff
@@ -46,6 +46,7 @@ case $# in
 	;;
 esac
 
+clone git github.com/coreos/go-systemd/daemon
```

Comment:
We are still missing the vendored code in this PR. could you run hack/vendor.sh for just this dependency?